cmd/notify-mailer/main.go

```diff
@@ -35,31 +34,34 @@ type mailer struct {
 	mailer        bmail.Mailer
 	subject       string
 	emailTemplate *template.Template
-	destinations  []recipient
+	recipients    []recipient
 	targetRange   interval
 	sleepInterval time.Duration
 }
 
-// interval defines a range of email addresses to send to, alphabetically.
-// The "start" field is inclusive and the "end" field is exclusive.
-// To include everything, set "end" to "\xFF".
+// interval defines a range of email addresses to send to, alphabetically. The
+// "start" field is inclusive and the "end" field is exclusive. To include
+// everything, set "end" to "\xFF".
 type interval struct {
 	start string
 	end   string
 }
 
-type contactJSON struct {
-	ID      int
+// queryResult is a gorp selectable holder for database query results.
+type queryResult struct {
+	// ID is exported to receive query return values.
+	ID int64
+
+	// Contact is exported to receive query return values.
 	Contact []byte
 }
 
 func (i *interval) ok() error {
 	if i.start > i.end {
 		return fmt.Errorf(
-			"interval start value (%s) is greater than end value (%s)",
+			"interval start value: %s is greater than end value: %s",
```

Comment:
hyper nit: I'm not a huge fan of this style personally. In general, my policy is "colons should only be in error messages to separate chains of wrapped (%w) error messages". This "key: value" style is great when there are long lists of k/v pairs separated by commas, but in the middle of an english sentence it makes it unclear where the sentence starts and stops.